### ISSUE TITLE
Allow the use of other permissions instead of accessibility

### ISSUE THREAD

**Issue by Etch9** [flxapps/DetoxDroid#116]
Accessibility is a privacy risk detox droid doesn't need the kind of power that accessibility gives, in some android ROMs (like grapheneOS) u cannot even give "Downloaded apps" that permission, it would be great to be able to grant more privacy focused permission that new android versions can offer.

**Comment by Van-QA** [comment]
In ‌some cases, we are unable to use other banking apps because of the ac‌cessibi‌lity permission enabled on DetoxDroi‌d.
While w‌aiting for the fix, do we have any workaround solution that help resolve the issue?

Thank you
